lib/foreman_openscap/engine.rb

```diff
@@ -1,6 +1,6 @@
 module ForemanOpenscap
   def self.with_katello?
-    defined?(::Katello)
+    Foreman::Plugin.installed?("Katello")
```

Comment:
Is the plugin really named 'Katello'? :thinking:

Reply:
I think it's `katello`

Reply:
yup, it is: https://github.com/Katello/katello/blob/4a363fd945d56ad48f6a30446d9c16320819d4a5/lib/katello/plugin.rb#L5

Reply:
just `katello` lowered case though :O